src/MountedTraversal.js

```diff
@@ -257,7 +257,7 @@ export function pathToNode(node, root) {
 }
 
 export function parentsOfInst(inst, root) {
-  return pathToNode(inst, root).reverse();
+  return (pathToNode(inst, root) || []).reverse();
```

Comment:
it seems to me that pathToNode should just always return an array instead of having this `|| []` logic.